tls/s2n_async_pkey.c

```diff
@@ -25,33 +25,6 @@
 #include "utils/s2n_result.h"
 #include "utils/s2n_safety.h"
 
-typedef enum { S2N_ASYNC_DECRYPT, S2N_ASYNC_SIGN } s2n_async_pkey_op_type;
-
-struct s2n_async_pkey_decrypt_data {
-    s2n_async_pkey_decrypt_complete on_complete;
-    struct s2n_blob                 encrypted;
-    struct s2n_blob                 decrypted;
-    unsigned                        rsa_failed : 1;
-};
-
```

Comment:
If you moved this to the header just to be able to check the operation type, I'm not sure that's a good idea. The op type will be accessible via a public API shortly anyway, and you could always just include the source like your comment on the include says.

Is there another reason this should be in the header?

Reply:
Just that one

Reply:
Done. Reverted this change